draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -295,28 +312,45 @@ steps, taking the `stream` as input.
 ### Parsing the index section {#index-section}
 
 The "index" section defines the set of HTTP requests in the bundle and
-identifies their locations in the "responses" section.
+identifies their locations in the "responses" section. It consists of a sequence
+of alternating request headers and response lengths:
 
 ~~~ cddl
-index = {* headers => [ offset: uint,
-                        length: uint] }
+index = [* (headers, length: uint) ]
 ~~~
 
-To parse the index section, given its `sectionContents`, the `sectionsStart`
-offset, the `sectionOffsets` CBOR item, and the `metadata` map to fill in,
-the parser MUST do the following:
+To parse the index section, given its `sectionContents`, the `sectionOffsets`
+map, and the `metadata` map to fill in, the parser MUST do the following:
 
 1. Let `index` be the result of parsing `sectionContents` as a CBOR item
    matching the `index` rule in the above CDDL ({{parse-cbor}}). If `index` is
    an error, return an error.
 
+1. Let `currentOffset` be a positive integer.
+
+1. Check that the responses array has the right number of items, and compute `currentOffset`:
```

Comment:
Should we scope this step (+its substeps) to parsing the responses array header?

Reply:
Sure.